sway/commands/allow_tearing.c

```diff
@@ -0,0 +1,25 @@
+#include <sway/commands.h>
+#include "sway/config.h"
+#include "sway/tree/view.h"
+#include "util.h"
+
+struct cmd_results *cmd_allow_tearing(int argc, char **argv) {
+  struct cmd_results *error = NULL;
```

Comment:
Style: indent contains spaces instead of tabs here (applies to elsewhere in the file as well)

Reply:
Fixed.

By default i use spaces as indentation in vscode, and when creating those new files, it used that setting without it really being noticeable 🙂